Resources/_template_week2_Haikus.md

```diff
@@ -0,0 +1,31 @@
+# Template for the haiku assignment
+
+## Instructions
+
+1. Form a subteam of three students from your team.
+2. As a subteam, create one branch on GitLab named `Group<groupnumber>/Haiku_<teamletter><subteamnumber>` (e.g. `Group1/Haiku_A1`).
+3. Copy this Template and rename it to the same name as the branch; keep the extension `.md` (e.g. `Group1/Haiku_A1.md`).
+4. One of the teammembers writes the first line of the haiku in the file on their local computer. Then uploads it to the branch.
```

Comment:
I think it helps if you use the git terminology: instead of upload, use pushes to the remote, and specify the "branch" is the subteams branch on remote.

Reply:
step 3, specify they should copy to the /Assignments subfolder, and do not touch the /Resources.